### ISSUE TITLE
Feature Request: +++ for new page :)

### ISSUE THREAD

**Issue by Scribilicious** [notaapp/nota#104]
Can we user **+++** to insert page breaks? Handy for when using markdown for printing docs.

Snippet for a page break in HTML:

`<div style="page-break-after: always;"></div>`

**Comment by Scribilicious** [comment]
I created a custom command for that... however would still be nice if it could be somehow visualized and support the +++ page break.

My snippet:
```
#!/bin/bash
# @parameters file
sed -i '' "s/+++/<div style=\"page-break-after:always\"><\/div>/g" $1$2
```

**Comment by astoilkov** [comment]
We implemented this. The next release will insert `page-break-after: always;` when writing `+++`. This will be useful when printing and exporting to PDF.

**Comment by Scribilicious** [comment]
Nice ❤️

**Comment by astoilkov** [comment]
This is now available in the 0.34 update.

Closing this issue. Thanks for the idea!
